src/NServiceBus.NHibernate.PersistenceTests/NServiceBus.NHibernate.PersistenceTests.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net461;netcoreapp3.1</TargetFrameworks>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\Test.snk</AssemblyOriginatorKeyFile>
```

Comment:
Did we actually need to introduce an additional snk file here? Can we just use NServiceBus.snk instead?

Reply:
afaik @lailabougria ran into an issue with one of the packages when she did use the regular NServiceBus.snk. I believe somewhere in the process the Conventions are involved and since every message type in assemblies signed with the NServiceBus.snk isn't considered as a message type this caused problems.